### DISCUSSION TITLE
Qdrant usage issues

### DISCUSSION THREAD

**Discussion by wanghaiyongyw** [qdrant/qdrant#3799] [General]
1. What is the indexing mechanism of qdrant, and does indexing consume a lot of memory?
2. Using nested retrieval can saturate the SSD disk I/O. Are there any relevant optimization configurations?
3. When deploying qdrant with Docker, restarting the disk directly on a normally used node will release a lot of space. What contents are being released?

**Comment by generall** [comment]
> What is the indexing mechanism of qdrant, and does indexing consume a lot of memory?

please refer to the docs: https://qdrant.tech/documentation/concepts/indexing/

> Using nested retrieval can saturate the SSD disk I/O. Are there any relevantt t optimization configurations?
 
Please elaborate what you mean by `nested retrieval`

> restarting the disk directly on a normally used node will release a lot of space.

Not sure I understand of what you mean by that
